src/WebUI/Pods/PodsTable.tsx

```diff
@@ -122,30 +113,22 @@ export class PodsTable extends BaseComponent<IPodsTableProperties> {
 
     private static _renderPodNameCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<V1Pod>, pod: V1Pod): JSX.Element {
         const textToRender = pod.metadata.name;
-        let colDataClass = css(colDataClassName, "primary-text");
-        const nameLabel = <span className={colDataClass}> {textToRender} </span>
-        const itemToRender = (
-            <ResourceStatus
-                statusProps={Utils.generatePodStatusProps(pod.status)}
-                customDescription={nameLabel}
-            />
-        );
-        return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender);
+        const statusProps: IResourceStatusProps = { statusProps: Utils.generatePodStatusProps(pod.status) }
+        const itemToRender = BaseKubeTable.renderColumn(textToRender, BaseKubeTable.defaultColumnRenderer, css(colDataClassName, "primary-text"));
+        return BaseKubeTable.renderTableCell(rowIndex, columnIndex, tableColumn, itemToRender, statusProps);
     }
 
     private static _renderPodWorkload(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<V1Pod>, pod: V1Pod): JSX.Element {
         const textToRender = pod.metadata.ownerReferences[0].name;
-        const itemToRender = BaseKubeTable.renderColumn(textToRender || "", BaseKubeTable.defaultColumnRenderer, colDataClassName);
-        //could not change link color to black hence commenting for now.
-        // const itemToRender: React.ReactNode = (
-        //     <Link
-        //         className=""
-        //         excludeTabStop
-        //         href="#"
-        //     >
-        //         {textToRender}
-        //     </Link>
-        // );
+        const itemToRender: React.ReactNode = (
+            <Link
+                className="fontSizeM text-ellipsis bolt-table-link bolt-table-inline-link"
```

Comment:
is there a better way, not to hard code these?
className

Reply:
@madhuv-msft I'll take it up in another pass.